spec/rdkafka/consumer_spec.rb

```diff
@@ -205,8 +208,6 @@ def send_one_message(val)
         expect(records&.payload).to eq "payload c"
         records = consumer.poll(timeout)
         expect(records).to be_nil
-
-        consumer.commit
```

Comment:
Removing the commit got the test passing. It was failing with `Local: No offset stored (no_offset)` which I'm guessing the seek+poll didn't cause the internal offset to be advance or it did but had already committed.